.github/workflows/ephemeral.yml

```diff
@@ -10,13 +10,31 @@ jobs:
       - uses: actions/checkout@v3
 
       - name: Deploy Preview
-        uses: ./preview
+        uses: jenseng/dynamic-uses@v1
         with:
-          localstack-api-key: ${{ secrets.LOCALSTACK_API_KEY }}
-          github-token: ${{ secrets.GITHUB_TOKEN }}
+          uses: LocalStack/setup-localstack@${{ env.action-version }}
+          with: |-
+            {
+              "github-token": "${{ secrets.GITHUB_TOKEN }}",
+              "state-backend": "preview",
```

Comment:
I'm just noticing the term `preview` here, from which I'd like to move away from (and rather employ the term `ephemeral`.

Reply:
Agree, however for backward compatibility I'd keep it `preview` for now.